onnx/__init__.py

```diff
@@ -240,63 +189,62 @@ def load_model(
 
 
 def load_tensor(
-    f: Union[IO[bytes], str],
-    format: Optional[Any] = None,  # pylint: disable=redefined-builtin
+    f: IO[bytes] | str,
+    format: _SupportedFormat = "protobuf",  # pylint: disable=redefined-builtin
 ) -> TensorProto:
     """Loads a serialized TensorProto into memory.
 
     Args:
         f: can be a file-like object (has "read" function) or a string containing a file name
-        format: for future use
+        format: The serialization format. Default is "protobuf".
 
     Returns:
         Loaded in-memory TensorProto.
     """
-    s = _load_bytes(f)
-    return load_tensor_from_string(s, format=format)
+    return load_tensor_from_string(_load_bytes(f), format)
 
 
 def load_model_from_string(
-    s: bytes,
-    format: Optional[Any] = None,  # pylint: disable=redefined-builtin
+    s: bytes | str,
+    format: _SupportedFormat = "protobuf",  # pylint: disable=redefined-builtin
 ) -> ModelProto:
     """Loads a binary string (bytes) that contains serialized ModelProto.
 
     Args:
         s: a string, which contains serialized ModelProto
-        format: for future use
+        format: The serialization format. Default is "protobuf".
+        encoding: The encoding of the bytes when format is a text format. Default is "utf-8".
```

Comment:
encoding parameter was removed but I would still mention that utf-8 is used. Encoding is usually not used for english languages but used for every other language.

Reply:
Done